src/components/BaseHeading.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const BaseHeading = ({ level, text }) => {
+  const HeadingTag = `h${level}`
+
+  return (
+    <HeadingTag className="text-[50px] md:text-[64px] text-center font-header font-bold">
```

Comment:
issue: Let's use default Tailwind values instead of the bracketed pixel values (ex: `text-[50px] md:text-[64px]`) Please push back with the designer if there doesn't exist a TW value.

Reply:
Is there a specific reason to use Tailwind values for font sizes? The end result is the same, and it might be more convenient to directly add the px unit in Figma when using Tailwind values.